src/einsteinpy/tests/test_symbolic/test_vaccum_metrics.py

```diff
@@ -0,0 +1,11 @@
+import numpy as np
+import sympy
+
+from einsteinpy.symbolic import MetricTensor, vaccum_metrics
+
+
+def schwarzschild_metric():
+    symbolstr = "t r theta phi"
+    syms = sympy.symbols(symbolstr)
+    sch = vaccum_metrics.SchwarzschildMetric(syms)
+    return sch
```

Comment:
Also, I'm giving you the test for `SchwarzschildMetric` this time, as discussed.
```
import numpy as np
import sympy

from einsteinpy.symbolic import ChristoffelSymbols, MetricTensor, vaccum_metrics


def test_ChristoffelSymbols():
    symbolstr = "t r theta phi"
    sch = vaccum_metrics.SchwarzschildMetric()
    chl = ChristoffelSymbols.from_metric(sch)
    mat = chl.tensor()
    syms = sympy.symbols(symbolstr)
    G, M, c, a = sympy.symbols("G M c a")
    assert (
        sympy.simplify(mat[2, 3, 3] - (-1 * sympy.cos(syms[2]) * sympy.sin(syms[2])))
        == 0
    )
    assert sympy.simplify(mat[3, 3, 1] - syms[1] / (syms[1] ** 2)) == 0
    assert (
        sympy.simplify(
            (mat[1, 1, 1].subs({a: (2 * G * M / (c ** 2))}))
            - (G * M / (2 * G * M * syms[1] - c ** 2 * syms[1] ** 2))
        )
        == 0
    )
    assert chl.symbols() == syms

```
Replace the whole file with the snippet, take care of the spelling change

Reply:
Before committing kindly run `black` & `isort`
Also run the tests on your local machine using `pytest -vv` to see whether all tests are running correctly, and if not, then single out the problems. Avoid committing faulty code.

Reply:
I have been taking care of all the comments you have mentionated above. 
Regarding to the formating of the code I used to use `tox -e reformat ` because I thought that I could use that command or `black & isort`, but please let me know in what cases should I tox or black, or if I should use both.

About the test, I agree about the importance of them, that is why I am kind of worry. I am really thankful about the examples you provide me but maybe it would be more useful for all of us if I learn how to do them. Maybe could you give me some documentation regarding to this topic?

Reply:
`tox` is like a combined command, which takes care of everything, but it is slow sometimes, whereas `black & isort` work instantly. Use any one of them. It just brings the code style to a specific standard. See the black code style https://black.readthedocs.io/en/stable/the_black_code_style.html . Just google for `isort`. 

Testing can sometimes seem very lame, as we are asserting very obvious logics, but sometimes we use testing to match results from our code to publicly available information.

Reply:
Okey I'm going to look at all the tests in detail and I will conitnue with tox. 
Can I make the commit of this changes? I guess the feature freeze will be over

Reply:
Sure! In fact, try to correct all the suggestions. There is no feature freeze in place as of now.